pages/linux/xfce4-screenshooter.md

```diff
@@ -0,0 +1,31 @@
+# xfce4-screenshooter
+
+> The XFCE4 screenshot tool.
+
+- Launch the screenshooter GUI:
+
+`xfce4-screenshooter`
+
+- Take a screenshot of the entire screen and launch the GUI to ask how to proceed:
+
+`xfce4-screenshooter --fullscreen`
+
+- Take a screenshot of the entire screen and save it in the specified directory:
+
+`xfce4-screenshooter --fullscreen --save {{path/to/directory}}`
+
+- Wait some time before taking the screenshot:
```

Comment:
```suggestion
- Wait for "s" seconds before taking the screenshot:
```

Reply:
Again, never saw this in any page and I feel like it is clear enough how I wrote it.

Reply:
I'm unsure that this needs changing, @mfrw. If you wanted to change it, you could do something like this, for example:

```suggestion
- Wait a specified amount of time before taking the screenshot:
```